llama_stack/templates/ollama/run-with-safety.yaml

```diff
@@ -32,6 +32,9 @@ providers:
   - provider_id: llama-guard
     provider_type: inline::llama-guard
     config: {}
+  - provider_id: code-scanner
```

Comment:
Are these be generated with distro_codegen.py? E.g. https://github.com/meta-llama/llama-stack/blob/ca66a1b188a64e96c84b280589e049b490a7fa9d/llama_stack/templates/meta-reference-gpu/meta_reference.py#L85

Reply:
Thanks for the pointer. Updated those, and ran `python3 llama_stack/scripts/distro_codegen.py` to make sure generated -with-safety files are in good state so the tests pass.